platforms/spring-boot/components-starter/camel-undertow-starter/src/main/java/org/apache/camel/component/undertow/springboot/UndertowComponentConfiguration.java

```diff
@@ -123,6 +123,10 @@ public void setResolvePropertyPlaceholders(
          * Set if the Undertow host should use direct buffers.
          */
         private Boolean directBuffers;
+        /**
+         * Set if the Undertow host should use http2 protocol.
+         */
+        private Boolean http2Enabled;
```

Comment:
Why adding the option on component level again? You already have it in undertowHostOptions.